package.json

```diff
@@ -24,60 +24,61 @@
     "watch": "npm run build:es -- --watch",
     "watch:lib": "npm run build:lib -- --watch",
     "watch:commonjs": "npm run build:commonjs -- --watch",
-    "test": "mocha -R spec ./test/unit/**",
+    "test": "mocha -r @babel/register -R spec ./test/unit/**",
     "test:cov": "nyc --reporter=lcov --reporter=html npm run test",
     "codecov": "cat coverage/lcov.info | codecov",
     "lint": "eslint src test",
     "lint:fix": "npm run lint -- --fix",
     "format": "prettier --write \"src/**/*.js\" \"test/**/*.js\"",
     "prepare": "npm run clean && npm run build",
-    "preversion": "npm run clean && npm run build",
+    "prepublish": "npm run clean && npm run build",
     "pre-push": "npm run lint"
   },
   "dependencies": {
-    "immer": "8.0.1",
-    "lodash": "^4.17.20",
+    "debug": "^4.3.1",
+    "immer": "9.0.1",
```

Comment:
There is a major version change to immer here - was that the intention? To keep this to just a minor version change we will need to understand the impact of this update